requirements-dev.txt

```diff
@@ -1,11 +1,11 @@
 -r requirements.txt
-fake-factory==0.5.0
-ipdb==0.9.3
-nsot==1.1.1
-py==1.4.26
-pytest==2.7.0
-pytest-django==2.9.1
-pytest-pythonpath==0.6
-Sphinx==1.3.6
-sphinx-autobuild==0.6.0
-sphinx-rtd-theme==0.1.9
+fake-factory~=0.5.0
+ipdb~=0.9.3
+nsot>=1.1.2,~=1.1.0
```

Comment:
My reasoning behind this is that it's okay if nsot is ahead by some patch-level versions since any API deprecations will take a few minor versions to happen (and this pins the minor version to `1.1`). 

Where I imagine us running into trouble is if the CLI gets ahead of the API and has some feature the API doesn't support yet, hence the `>=1.1.2` requirement in there as well.

That was my reasoning anyway. I'm down to pin it to a specific version of nsot but I felt like that wasn't necessary anymore.